### ISSUE TITLE
`FlxSprite.setGraphicSize` only takes integers

### ISSUE THREAD

**Issue by Starmapo** [HaxeFlixel/flixel#2878]
Is there a reason why `FlxSprite.setGraphicSize` only takes integers? The `width` and `height` properties for `FlxObject` are floats, I don't see why the function is limited to integers. For example, it's a little annoying having to do:
```haxe
sprite.setGraphicSize(Std.int(otherSprite.width), Std.int(otherSprite.height));
```
rather than just:
```haxe
sprite.setGraphicSize(otherSprite.width, otherSprite.height);
```
Even if the width and height aren't decimal numbers, it prevents having to do `Std.int` twice.

**Comment by Starmapo** [comment]
I notice replacing `width` and `height` with a collider for objects is planned (#2716), which I imagine results in separate properties to get the graphic size. Will those be integers? An image's size can't have decimals after all. If so, `setGraphicSize` could remain using integers:
```haxe
sprite.setGraphicSize(otherSprite.graphicWidth, otherSprite.graphicHeight);
```

**Comment by Geokureli** [comment]
> An image's size can't have decimals after all. If so, `setGraphicSize` could remain using integers:
> 
> ```hx
> sprite.setGraphicSize(otherSprite.graphicWidth, otherSprite.graphicHeight);
> ```

graphicWidth and graphicHeight will be floats as well, so it really does make sense to change `setGraphicSize` to take floats

**Comment by Starmapo** [comment]
Alright, I'll be making a PR

**Comment by Geokureli** [comment]
keep open, when you make a PR put `closes #2878` and it will close this when merged, thanks!
